commit 25186731fa1d0021e61f630fa393ec3c9b3867c4 (refs/changes/43/5443/3)
Author: sjorge <sjorge@blackdot.be>
Date:   2019-01-29T15:55:59+01:00 (8 months ago)
    
    OS-7545 incorrect update_disks example in vmadm manpage
    Reviewed by: John Levon <john.levon@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/src/vm/man/vmadm.1m.md b/src/vm/man/vmadm.1m.md
index 47de1896..72b030fa 100644
--- a/src/vm/man/vmadm.1m.md
+++ b/src/vm/man/vmadm.1m.md
@@ -450,11 +450,13 @@ tab-complete UUIDs rather than having to type them out for every command.
         been larger.
 
          {
-           "update_disks": {
-             "path": "/dev/zvol/rdsk/zones/.../disk1",
-             "size": 10,
-             "dangerous_allow_shrink": true
-           }
+           "update_disks": [
+             {
+               "path": "/dev/zvol/rdsk/zones/.../disk1",
+               "size": 10,
+               "dangerous_allow_shrink": true
+             }
+           ]
          }
 
         Those fields marked in the PROPERTIES section below as updatable and
